test/unit/requester-core.test.js

```diff
@@ -3,6 +3,8 @@ var expect = require('chai').expect,
     runtimeVersion = require('../../package').version,
     requesterCore = require('../../lib/requester/core');
 
+const IS_NODE = typeof window === 'undefined';
```

Comment:
in tests, when will this be false ?

Reply:
For browser tests, this will be false. All the other tests this will be true.